test/wasmex/memory_test.exs

```diff
@@ -141,12 +137,10 @@ defmodule Wasmex.MemoryTest do
       Wasmex.Memory.set(memory, 3, 108)
       # o
       Wasmex.Memory.set(memory, 4, 111)
-      # automatically added null byte
-      Wasmex.Memory.set(memory, 5, 0)
 
-      assert Wasmex.Memory.read_binary(memory, 0) == "hello"
-      assert Wasmex.Memory.read_binary(memory, 3) == "lo"
-      assert Wasmex.Memory.read_binary(memory, 8) == ""
+      assert Wasmex.Memory.read_binary(memory, 0, 5) == 'hello'
+      assert Wasmex.Memory.read_binary(memory, 3, 2) == 'lo'
+      assert Wasmex.Memory.read_binary(memory, 8, 0) == <<>>
```

Comment:
Thanks for updating the tests! Reading them, I think having a `read_string` method that pipes `read_binary` into `to_string()` would be nice to have. 

My long-term goal is to come close to the usability of `wasm-bindgen` for JavaScript. At least as long as it is not too hard to maintain the beast we create that way :) convenient string handling would be a great step in that direction.